sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -108,15 +110,15 @@ export interface JsonWebKey {
 
 /**
  * @interface
- * An interface representing the complete key with value
+ * An interface representing a KeyVault Key, with its name, value and {@link KeyProperties}.
  */
 export interface KeyVaultKey {
   /**
    * @member {string} [key] The key value.
    */
   key?: JsonWebKey;
   /**
-   * @member {string} [name] The name of key/secret/certificate.
+   * @member {string} name The name of the KeyVault Key.
```

Comment:
```suggestion
   * @member {string} name The name of the key.
```

Reply:
I'm not seeing the value in this change.